tests/sentry/rules/conditions/test_event_attribute.py

```diff
@@ -57,6 +58,14 @@ def get_event(self, **kwargs):
                     "crash_type": "crash",
                 },
             },
+            "threads": {
+                "values": [
+                    {
+                        "id": 1,
+                        "main": True,
+                    },
+                ],
+            },
```

Comment:
I don't know if I should use the `threads` and `thread_id` here or the `exception_main_thread` out of the `data` property.
Is this JSON the representation of the Snuba/Kafka message or the event itself? how should I test this?

Reply:
I believe the JSON is the event itself. tbh, I'm not sure the `event.snuba_data` property will be hydrated when this rule gets evaluated. I would just use the thread interface directly.